src/modules/http.js

```diff
@@ -92,7 +96,13 @@ module.exports = class Http {
       strict_variables: false
     });
 
-    app.use(express.urlencoded({ limit: '12mb', extended: true, parameterLimit: 50000 }));
+    app.use(
+      express.urlencoded({
+        limit: '12mb',
+        extended: true,
+        parameterLimit: 50000
+      })
+    );
```

Comment:
I am not a big fan of refactoring code if it doesn't have anything to do with the feature.
Please create another pull request for this

Reply:
I think this might be the `prettier` plugin